### ISSUE TITLE
rank of modules over quotient rings

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#312]
Something's wrong here, since 20 > 6, and
```
S = ZZ/101[x_0..x_4];
I = ideal(x_0^5 + x_1^5 + x_2^5 + x_3^5 + x_4^5 + 2*x_0*x_1*x_2*x_3*x_4);
R = S/I;
M = cokernel map((R)^{{ -2},{ -2},{ -1},{ -1},{ -1},{ -1}},(R)^{{ -4},{ -4}},{{4*x_3^2, x_0^2+10*x_2^2}, {9*x_0^2+x_2^2, x_1^2}, {8*x_2^3, 9*x_2^3}, {2*x_3^3, 7*x_3^3}, {4*x_1^3, 9*x_1^3+x_2^3}, {9*x_1^3, x_1^3+7*x_4^3}});
rank M
```
yields
```

i1 : S = ZZ/101[x_0..x_4];

i2 : I = ideal(x_0^5 + x_1^5 + x_2^5 + x_3^5 + x_4^5 + 2*x_0*x_1*x_2*x_3*x_4);

o2 : Ideal of S

i3 : R = S/I;

i4 : M = cokernel map((R)^{{ -2},{ -2},{ -1},{ -1},{ -1},{ -1}},(R)^{{ -4},{ -4}},{{4*x_3^2, x_0^2+10*x_2^2}, {9*x_0^2+x_2^2, x_1^2}, {8*x_2^3, 9*x_2^3}, {2*x_3^3, 7*x_3^3}, {4*x_1^3, 9*x_1^3+x_2^3}, {9*x_1^3, x_1^3+7*x_4^3}});

i5 : rank M

o5 = 20
```

**Comment by DanGrayson** [comment]
fixed in commit 60d46e30d86199f07a6af1e1b25a34dd4ea0adee

**Comment by DanGrayson** [comment]
```

i1 : R = ZZ/101[x_0..x_4]/(x_0^5 + x_1^5 + x_2^5 + x_3^5 + x_4^5 + 2*x_0*x_1*x_2*x_3*x_4);

i2 : M = cokernel map((R)^{{ -2},{ -2},{ -1},{ -1},{ -1},{ -1}},(R)^{{ -4},{ -4}},{{4*x_3^2, x_0^2+10*x_2^2}, {9*x_0^2+x_2^2, x_1^2}, {8*x_2^3, 9*x_2^3}, {2*x_3^3, 7*x_3^3}, {4*x_1^3, 9*x_1^3+x_2^3}, {9*x_1^3, x_1^3+7*x_4^3}});

i3 : rank M

o3 = 4
```
